operator/identitygc/crd_gc.go

```diff
@@ -170,9 +170,11 @@ func (igc *GC) gc(ctx context.Context) error {
 	if ctx.Err() == nil {
 		igc.successfulRuns++
 		igc.metrics.IdentityGCRuns.WithLabelValues(LabelValueOutcomeSuccess).Set(float64(igc.successfulRuns))
+		igc.metrics.IdentityGCLatency.Set(float64(time.Since(timeNow).Milliseconds()))
```

Comment:
Is there a reason to use milliseconds here? In the code base, we have more usages of seconds and no usages of milliseconds:

```
❯ rg "\.Set(.+Seconds)"
clustermesh-apiserver/syncstate/syncstate.go
32:             metrics.BootstrapDuration.WithLabelValues(clusterInfo.Name).Set(syncTime.Seconds())

enterprise/plugins/hubble-flow-export/metrics_plugin.go
156:    h.flowsExportTimestamp.Set(float64(flow.GetTime().GetSeconds()))

vendor/google.golang.org/protobuf/encoding/protojson/well_known_types.go
630:    m.Set(fdSeconds, protoreflect.ValueOfInt64(secs))
819:    m.Set(fdSeconds, protoreflect.ValueOfInt64(secs))

pkg/rate/metrics/metrics.go
23:     metrics.APILimiterWaitDuration.WithLabelValues(name, "max").Set(v.MaxWaitDuration.Seconds())
24:     metrics.APILimiterWaitDuration.WithLabelValues(name, "min").Set(v.MinWaitDuration.Seconds())

pkg/k8s/watchers/pod.go
330:                    metrics.EventLagK8s.Set(timeSinceEpCreated.Round(time.Second).Seconds())
```